.github/workflows/ci.yml

```diff
@@ -5,15 +5,18 @@ name: build
 
 on:
   push:
-    branches:
-      - develop
+    branches: [develop, 'GEN*', 'gen*']
```

Comment:
Actually could you also add `main` to the branches here? I don't know if your current setup for ghcr will always create a docker image that is based on the latest version of the repo like we have for dockerhub? Will this setup create a `latest` tag by default?